ouroboros-network-protocols/test-cddl/specs/common.cddl

```diff
@@ -21,8 +21,11 @@ origin          = []
 blockHeaderHash = [slotNo, int]
 slotNo          = word64
 
-txId         = int
-transaction  = int
+; In this spec we don't specify what transaction idenfiers (txId) or
+; transactions (tx) are themselves, `ouroboros-network` is polymorphic over
+; them.
+txId         = any
+tx           = any
```

Comment:
Where can I find specification of what is expected here by the actual cardano-node implementation?

Reply:
@coot